src/session.ts

```diff
@@ -52,6 +49,15 @@ export const enum types {
   ReadWrite = 'readwrite',
 }
 
+export type GetSessionMetadataCallback = ResourceCallback<
+  spanner_client.spanner.v1.ISession,
+  r.Response
```

Comment:
Remove

1. Only one value is returned as per above
2. `r.Response` should never be used as per Dave's [comment](https://github.com/googleapis/nodejs-spanner/pull/627#pullrequestreview-246182206), point one